### PR TITLE
https://issues.redhat.com/browse/ACM-8367 Search operator known issues

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8367

### CODE REVIEW

File: release_notes/known_issues_observability.adoc [hunk 1]
```diff
@@ -27,6 +27,30 @@ For your {ocp} cluster, see link:https://access.redhat.com/documentation/en-us/o
...
+----
+
+. Update the `search-postgres` deployment with a limit for the `hugepages` parameter. Run the following command to set the `hugepages` parameter to `512Mi`:
++
+[source,bash]
+----
+oc patch deployment search-postgres --type json -p '[{"op": "add", "path": "/spec/template/spec/containers/0/resources/limits/hugepages-2Mi", "value":"512Mi"}]'
+----
+
+//Verification step needed
```
Comment (dockerymick) [inline release_notes/known_issues_observability.adoc]: @jlpadilla can you help here please

**Review by zlayne — APPROVED** [review]

File: release_notes/known_issues_console.adoc [hunk 2]
```diff
@@ -27,6 +27,30 @@ For your {ocp} cluster, see link:https://access.redhat.com/documentation/en-us/o
...
+----
+
+. Update the `search-postgres` deployment with a limit for the `hugepages` parameter. Run the following command to set the `hugepages` parameter to `512Mi`:
++
+[source,bash]
+----
+oc patch deployment search-postgres --type json -p '[{"op": "add", "path": "/spec/template/spec/containers/0/resources/limits/hugepages-2Mi", "value":"512Mi"}]'
+----
+
+//Verification step needed
```
Comment (dockerymick) [inline release_notes/known_issues_console.adoc]: @zlayne thanks for your review and approval, will you help with adding the verification step if there is one? Will the user need to run a command to check the status ?

**Review by jc-berger — APPROVED** [review]
/lgtm

File: release_notes/known_issues_console.adoc (hunk 2 above)
Comment (dockerymick) [reply release_notes/known_issues_console.adoc]: @SherinV Will you help provide a verification step please?

Comment (SherinV) [reply release_notes/known_issues_console.adoc]: The postgres pod should be running and should have the hugepages set as above.
```
NAMESPACE=open-cluster-management
POSTGRES_POD=$(oc get pods -n ${NAMESPACE} | grep search-postgres | awk '{print $1}')
oc get po $POSTGRES_POD  -o jsonpath='{.spec.containers[0].resources.limits.hugepages-2Mi}'
```

This should show 512 Mi.

Comment (dockerymick) [reply release_notes/known_issues_console.adoc]: @SherinV thanks for your feedback. So the user must run the following command to verify the memory usage of the `search-postgres` pod: `oc get pods -n ${NAMESPACE} | grep search-postgres | awk '{print $1}')
oc get po $POSTGRES_POD  -o jsonpath='{.spec.containers[0].resources.limits.hugepages-2Mi}'`

will you confirm?

File: release_notes/known_issues_console.adoc [hunk 3]
```diff
@@ -27,6 +27,37 @@ For your {ocp} cluster, see link:https://access.redhat.com/documentation/en-us/o
...
+[source,bash]
+----
+oc patch deployment search-postgres --type json -p '[{"op": "add", "path": "/spec/template/spec/containers/0/resources/limits/hugepages-2Mi", "value":"512Mi"}]'
+----
+
+. Run the following command to verify the memory usage of the `search-postgres` pod:
++
+[source,bash]
+----
+oc get pods -n ${NAMESPACE} | grep search-postgres | awk '{print $1}') oc get po $POSTGRES_POD  -o jsonpath='{.spec.containers[0].resources.limits.hugepages-2Mi}'
```
Comment (dockerymick) [inline release_notes/known_issues_console.adoc]:
```suggestion
oc get pod <your-postgres-pod-name> -o jsonpath='{.spec.containers[0].resources.limits.hugepages-2Mi}'
```

File: release_notes/known_issues_console.adoc [hunk 4]
```diff
@@ -27,6 +27,37 @@ For your {ocp} cluster, see link:https://access.redhat.com/documentation/en-us/o
...
+----
+
+. Update the `search-postgres` deployment with a limit for the `hugepages` parameter. Run the following command to set the `hugepages` parameter to `512Mi`:
++
+[source,bash]
+----
+oc patch deployment search-postgres --type json -p '[{"op": "add", "path": "/spec/template/spec/containers/0/resources/limits/hugepages-2Mi", "value":"512Mi"}]'
+----
+
+. Run the following command to verify the memory usage of the `search-postgres` pod:
```
Comment (dockerymick) [inline release_notes/known_issues_console.adoc:59]:
```suggestion
. Before you verify the memory usage for the pod, make sure your `search-postgres` pod is in the `Running` state. Run the following command:
+
[source,bash]
----
oc get pod <your-postgres-pod-name>  -o jsonpath="Status: {.status.phase}"
----

. Run the following command to verify the memory usage of the `search-postgres` pod:
```

**Review by SherinV — APPROVED** [review]

**Comment by dockerymick** [comment]
@SherinV thanks for the review and approval during the live call. I appreciate you ! Merging this PR

**Pull request merged**